runtime/criusupport/criusupport.cpp

```diff
@@ -36,6 +37,8 @@
 
 extern "C" {
 
+typedef void (*fn_criu_set_unprivileged)(bool unprivileged);
```

Comment:
is it defined with `bool` in the criu API?

Reply:
Yes. I actually added the function since it is missing from https://github.com/checkpoint-restore/criu/pull/1155 however other functions taking a boolean (e.g. `criu_set_shell_job`) also define their booleans as `bool`.